docs/api/proxy-based-tracking.md

```diff
@@ -0,0 +1,145 @@
+---
+id: proxy-based-tracking
+title: Proxy-based Tracking
+sidebar_label: Proxy-based Tracking
+hide_title: true
+---
+
+# Proxy-based Tracking
+
+This document describes about `useTrackedState` hook.
```

Comment:
This line isn't necessary, especially if we move this content into a separate `useTrackedState` header section in the "Hooks" page.